src/App.js

```diff
@@ -25,6 +26,7 @@ function App() {
           element={
             <>
               <FullArticleCard />
+              <CommentList />
```

Comment:
Instead of loading the component in here I would load it from within the FullArticleCard component.